jwst/guider_cds/guider_cds.py

```diff
@@ -24,17 +34,29 @@ def guider_cds(model):
     rate is then given by the minimum of these two values for each pixel,
     divided by the group time.  For the FINEGUIDE mode, the count rate is the
     average of the last 4 groups minus the average of the first 4 groups,
-    divided by the group time.
+    divided by the group time.  The variances are calculated using gain and
+    read noise values obtained from reference files if possible, otherwise,
+    default representative constant values are used. From the variances the ERR
+    array is calculated.
 
     Parameters
     ----------
     model: data model
         input data model, assumed to be of type GuiderRawModel
-    """
 
+    gain_model: instance of gain Model or None
```

Comment:
```suggestion
    gain_model : `datamodels.GainModel` or None
```

Reply:
Note that for the docstrings we specify the `type` of object, not a description of a type. Also there should be a space between the parameter name and `:`